docs/source/getting_started/quick_start.md

```diff
@@ -0,0 +1,64 @@
+# 🚀 Quick Start
+
+Get up and running with Straming in a couple of steps!
+
+First, convert your raw dataset into a supported streaming format, for example, `mds` format.  After the conversion is complete, upload your streaming dataset to the cloud based storage of your choice (e.g., [AWS S3](https://aws.amazon.com/s3/)).
+
+```python
+import numpy as np
+from PIL import Image
+from uuid import uuid4
+from streaming import MDSWriter
+
+# Directory path to store the output compressed files
+dirname = 'dirname'
+
+# A dictionary of input fields to an Encoder/Decoder type
+columns = {
+    'uuid': 'str',
+    'img': 'jpeg',
+    'clf': 'int'
+}
+
+# Compression algorithm name
+compression = 'zstd'
+
+# Hash algorithm name
+hashes = 'sha1', 'xxh64'
+
+# Generates random images and classes for input sample
+samples = [
+    {
+        'uuid': str(uuid4()),
+        'img': Image.fromarray(np.random.randint(0, 256, (32, 48, 3), np.uint8)),
+        'clf': np.random.randint(10),
+    }
+    for _ in range(1000)
+]
+
+# Call `MDSWriter` to iterate through the input data and write into a shard `mds` file
+with MDSWriter(dirname, columns, compression, hashes) as out:
+    for sample in samples:
+        out.write(sample)
+```
```

Comment:
@karan6181 would it make sense to also import `boto3` in this example and just do the upload to an S3 bucket from `dirname` as a part of this script?

Reply:
I think it is a good idea. But few usecase we need to consider here
1. Boto3 s3 requires `aws_access_key_id`, `aws_secret_access_key`, and `region_name` to instantiate an s3 client.
2. Boto3 s3 doesn't have an API to upload folder. It has `upload_file()` method which upload only single file. So we need to provide a method which iterate overate a directory and upload all the files. All of this will be a big code.

On the flip side, we can refer the user to use [AWS CLI](https://aws.amazon.com/cli/) to upload the data files. It also support to upload a folder. For example,

```
$ aws s3 cp dirname s3://mybucket/myfolder --recursive
```

I have added [AWS CLI](https://aws.amazon.com/cli/)  step in new `user_guide.md` file.

Let me know your thoughts. We can do the same in `quick_start.md` guide.

Reply:
As per offline discussion, updated the `quick_start.md` guide with using AWS CLI.